packages/experimental/Tabs/src/Tabs.styling.ts

```diff
@@ -0,0 +1,38 @@
+import { tabsName, TabsTokens, TabsSlotProps, TabsProps } from './Tabs.types';
+import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { fontStyles } from '@fluentui-react-native/tokens';
+
+import { defaultTabsTokens } from './TabsTokens';
+
+export const stylingSettings: UseStylingOptions<TabsProps, TabsSlotProps, TabsTokens> = {
+  tokens: [defaultTabsTokens, tabsName],
+  slotProps: {
+    root: buildProps(
+      () => ({
+        style: {
+          display: 'flex',
+          minHeight: 32,
+          minWidth: 80,
+        },
+      }),
+      [],
+    ),
+    label: buildProps(
+      (tokens: TabsTokens, theme: Theme) => ({
+        variant: 'subheaderSemibold',
```

Comment:
I don't think placing the default variant here is wrong but I would expect to see it in TabsTokens because that's where all the other default tokens are defined.